lib/driver.js

```diff
@@ -143,6 +142,16 @@ class AndroidUiautomator2Driver extends BaseDriver {
       };
       _.defaults(this.opts, defaultOpts);
 
+      if (this.isChromeSession) {
+        logger.info("We're going to run a Chrome-based session");
+        let {pkg, activity} = helpers.getChromePkg(this.opts.browserName);
+        this.opts.appPackage = pkg;
+        this.caps.appPackage = pkg;
+        this.opts.appActivity = activity;
+        this.opts.appActivity = activity;
```

Comment:
Yes. I didn't know this was going to be merged right away. Anyway, I have a few more fixes to try out, so I'll fix this there.